### PR TITLE
configure typescript

### PR DESCRIPTION
**Pull request by brianmuks**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "sparked-next",
+  "name": "next-typescript-example",
```
Comment (OlivierJM) [inline package.json:2]: Better to keep the name as is.

**Review by OlivierJM — CHANGES_REQUESTED** [review]
Let's also keep the package-lock.json for package consistency.

File: package.json (hunk 1 above)
Comment (brianmuks) [reply package.json:2]: I didn’t notice. Sorry I will revert

**Comment by brianmuks** [comment]
@OlivierJM . I have made the requested changes.

**Review by OlivierJM — APPROVED** [review]

**Pull request merged**